src/nrtk_explorer/library/yaml_transforms.py

```diff
@@ -152,6 +152,6 @@ def execute(self, input, *input_args):
             input_args = self.exec_args
 
         input_array = np.asarray(input)
-        output_array = self._perturber.perturb(input_array, *input_args)
+        output_array, unknown = self._perturber.perturb(input_array, *input_args)
```

Comment:
Very minor nitpick, I think the convention is to use `_` for unused elements when unpacking tuples:
```
output_array, _ = ...
```